### PR TITLE
Add right autoload path for Pimcore/Tests in documentation / Remove non-existing Functional Helper

### LINKED ISSUE #17004: [Improvement]: Insert right autoload path in Application Testing Documentation
### Description

In the Application Testing section of the Pimcore documentation there is a discrepancy between the _bootstrap.php and the functional.suite.yml code examples.
The autoloader for the first has the following path 
``` 
PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support'
``` 
mapped to:
``` 
'Pimcore\Tests'
``` 

But then in the functional.suite.yml example the Helper is called like this:
``` 
\Pimcore\Tests\Support\Helper\Pimcore:
``` 
Instead of this:
``` 
\Pimcore\Tests\Helper\Pimcore:
``` 

which is a small detail but can save a lot of time and research for copy pasters like myself.

Generally I would prefer to have 'Pimcore\Tests' mapped to PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests instead of PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support'

### Suggestions

I would also like to suggest removing 
``` 
- \Tests\Support\Helper\Functional:
``` 
From the functional.suite.yml code example since it does not exist and is not being generated. 

And I think functional.suite.yml should be renamed to **Functional.suite.yml** as it being generated and mentioned in the Codeception documentation.

### Sources
- Pimcore Application Testing Documentation:
``` 
https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Testing/Application_Testing
``` 
- Codeception Functional Suite Documentation:
``` 
https://codeception.com/docs/FunctionalTests
``` 

### Screenshots
Bildschirmfoto 2024-04-30 um 11 52 19

Bildschirmfoto 2024-04-30 um 11 51 21

### PR DESCRIPTION
**Pull request by sl0wlydeadly**
Add right autoload path for Pimcore/Tests

  
  

## Changes in this pull request  
Resolves #17004

### CODE REVIEW

File: doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ require_once PIMCORE_PROJECT_ROOT . '/vendor/autoload.php';
 // but is done here for demonstration purpose
 require_once PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support/Util/Autoloader.php';
 
-Autoloader::addNamespace('Pimcore\Tests', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support');
+Autoloader::addNamespace('Pimcore\Tests', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests');
```
Comment (kingjia90) [inline doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md]: Thank you for the PR!

By looking at
https://github.com/pimcore/data-hub/blob/7652b48a906f466cba4c2aa1d6e44d5cf4d59e7b/tests/_bootstrap.php#L52-L53
for Pimcore 11, it should be like
```suggestion
Autoloader::addNamespace('Pimcore\Tests\Support', PIMCORE_PROJECT_ROOT . '/vendor/pimcore/pimcore/tests/Support');
```

File: doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md [hunk 2]
```diff
@@ -355,7 +355,6 @@ suite to use the `\Pimcore\Tests\Support\Helper\Pimcore` helper:
 actor: FunctionalTester
 modules:
     enabled:
-        - \Tests\Support\Helper\Functional
```
Comment (kingjia90) [inline doc/19_Development_Tools_and_Details/29_Testing/01_Application_Testing.md]: i believe it's a placeholder for example purposes, the entire FunctionalTester(`functional.suite.yml`) is not set nor exists in the repository

**Comment by kingjia90** [comment]
mhhh, could you please open a new PR with only (see the resolved comments)https://github.com/pimcore/pimcore/pull/17130/commits/ee580a2341ca80566072943bdcb5186cecc1f1ba ? There are some conflicts caused by CS Fixer
Thank you in advance

**Comment by jdreesen** [comment]
> There are some conflicts caused by CS Fixer

The CS Fixer bot destroyed the PR? :astonished:

**Comment by sl0wlydeadly** [comment]
Hey. I am on it.

**Comment by sl0wlydeadly** [comment]
@kingjia90 i opened a new PR: https://github.com/pimcore/pimcore/pull/17598
This could be closed i think.

**Pull request closed without merging**